docs/reference/aggregations/bucket/terms-aggregation.asciidoc

```diff
@@ -252,13 +247,17 @@ GET /_search
 }
 --------------------------------------------------
 
-WARNING: Test any sorts on sub-aggregations before using them in production.
-Sorting on a sub-aggregation may return errors or inaccurate results. For
-example, due to the way the `terms` aggregation
+===== Ordering by a sub aggregation
+
+WARNING: Sorting by a sub aggregation generally produces incorrect ordering, due to the way the `terms` aggregation
 <<search-aggregations-bucket-terms-aggregation-shard-size,gets results from
-shards>>, sorting on a `max` sub-aggregation in _ascending_ order often produces
-inaccurate results. However, sorting on a `max` sub-aggregation in _descending_
-order is typically safe.
+shards>>.
+
+There are two cases when sub-aggregation sorting is safe.  Sorting by a maximum
+in descending order, or a minimum in ascending order will be correct.  This
+works only because the sort order aligns with the behavior of the sub
+aggregations, i.e. sorting maxes by largest value first or mins by lowest value
+first.
```

Comment:
```suggestion
There are two cases when sub-aggregation sorting is safe and returns correct results:
sorting by a maximum in descending order, or a sorting by a minimum in ascending order. 
Because the sort order aligns with the behavior of the sub aggregations, you can sort
`max` buckets by largest value first or `min` buckets by lowest value first.

```

Reply:
I think that you meant `max` and `min` buckets, but I might have misinterpreted your intention. Let me know if I'm off base here.

Reply:
Tricky, indeed! Here's another rendering that hopefully is more accurate:

> There are two cases when sub-aggregation ordering is safe and returns correct results: sorting by a maximum in descending order, or sorting by a minimum in ascending order. These approaches work because they align with the behavior of sub aggregations. That is, if you're looking for the largest maximum or the smallest minimum, the global answer (from combined shards) must be included in one of the local shard answers. Conversely, the smallest maximum and largest minimum wouldn't be accurately computed.

Reply:
Thanks, that reads pretty well to me.  I'll make the change.